src/FunctionApps/python/FhirServerExport/__init__.py

```diff
@@ -13,6 +13,10 @@ def main(req: func.HttpRequest) -> func.HttpResponse:
     poll_step = float(config.get_required_config("FHIR_EXPORT_POLL_INTERVAL", 30))
     poll_timeout = float(config.get_required_config("FHIR_EXPORT_POLL_TIMEOUT", 300))
 
+    container = config.get_required_config("FHIR_EXPORT_CONTAINER", "fhir-exports")
+    if container == "<none>":
```

Comment:
Is there a reason for setting the empty value to `"<none>"` as opposed to just `""`, which is what you set it to in the code?

Reply:
Yes, that would have been my preference and I did test the following approach:
* Un-set maps to the default fhir-exports container name
* Specifying an empty string maps to no container

Unfortunately, our `get_required_config` implementation doesn't differentiate between an un-set variable, and empty string (at least when run locally).  Both end up returning the default value (fhir-exports).  Rather than figuring out a change to `get_required_config` to this PR I decided to work around it by introducing the `<none>` special value.